src/main/java/core/basesyntax/Run.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Run {
```

Comment:
Same, let's remove this class from PR (and result.csv, result2.csv files)

Reply:
ok, done